src/dnlib.csproj

```diff
@@ -32,10 +33,11 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
 
-    <PackageDescription>Reads and writes .NET assemblies and modules, Windows PDBs and Portable PDBs.
```

Comment:
Undo the formatting. Whatever tool you're using, it does things it shouldn't be doing.

Reply:
This is Visual Studio 15.9.7 default formatting